### ISSUE TITLE
Fedora 25 - wok tab not redenering

### ISSUE THREAD

**Issue by aaronmailageek** [kimchi-project/wok#219]
Login into kimchi > Click on the Wok tab > neither of the sub tabs (activity log, settings) render.
![screen shot 2017-06-27 at 12 03 56](https://user-images.githubusercontent.com/29729776/27584679-c0573a70-5b30-11e7-8cbf-51e4cc6073a4.png)
![screen shot 2017-06-27 at 12 04 07](https://user-images.githubusercontent.com/29729776/27584680-c05dd844-5b30-11e7-8294-e86e7cc7b3e3.png)

Tried in multiple browsers (Chrome, Safari, FireFox, I.E 11).

Wok Version: 2.4.0-0

**Comment by alinefm** [comment]
Hi @aaronmailageek 

Could you check the error using Firebug or other web debugger tool? And post the output here

**Comment by aaronmailageek** [comment]
`VM2002:1 GET https://my-url:8001/tabs/user-log.html 500 (Internal Server Error)
(anonymous)	@	VM2002:1
send	@	jquery.min.js:5
ajax	@	jquery.min.js:5
m.fn.load	@	jquery.min.js:5
loadPage	@	wok.min.js:2327
onWokRedirect	@	wok.min.js:2318
j	@	jquery.min.js:2
fireWith	@	jquery.min.js:2
fire	@	jquery.min.js:2
updatePage	@	wok.min.js:2355
`

From the command line:
`[root@urvm03 tabs]# ll
total 16
-rw-r--r--. 1 root root 4405 Mar 29 17:23 settings.html.tmpl
-rw-r--r--. 1 root root 3043 Mar 29 17:23 user-log.html.tmpl
-rw-r--r--. 1 root root 3031 Mar 29 17:23 user-log-search.html.tmpl
[root@urvm03 tabs]# pwd
/usr/share/wok/ui/pages/tabs`

**Comment by aaronmailageek** [comment]
@alinefm  WorkAround:
```
cp /usr/share/locale/en_US/LC_MESSAGES/gingerbase.mo  /usr/share/locale/en_GB/LC_MESSAGES/
cp /usr/share/locale/en_US/LC_MESSAGES/kimchi.mo  /usr/share/locale/en_GB/LC_MESSAGES/
cp /usr/share/locale/en_US/LC_MESSAGES/wok.mo  /usr/share/locale/en_GB/LC_MESSAGES/

```
log out and back in to the WebUI
Relates to:
[https://github.com/kimchi-project/wok/issues/151](url)
[https://github.com/kimchi-project/wok/issues/192](url)

**Comment by alinefm** [comment]
Closing this issue as it as fixed by #151
